string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,10 @@
+/* exported isVowel */
+function isVowel(char) {
+  const vowels = ['a', 'e', 'i', 'o', 'u'];
+  for (let i = 0; i < vowels.length; i++) {
+    if (vowels[i] === char.toLowerCase()) {
+      return true;
+    }
+  }
+  return false;
+}
```

Comment:
This is an example of where a `switch` statement would be useful. Your solution totally works, and is fine, but just wanted to offer an alternative.
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```